assets/js/theme/common/carousel/utils/setupAriaAttributes.js

```diff
@@ -0,0 +1,11 @@
+export default ($node, ariaAttribute, attributeValue, ...accessibiltyClassList) => {
```

Comment:
Could we changes spread with selector?

Reply:
removed setupAriaAttributes util at all.